widget/embedded/src/pages/AddCustomTokenPage.tsx

```diff
@@ -178,6 +179,9 @@ export function AddCustomTokenPage() {
             <Divider size={10} />
 
             <Button
+              id={`widget-add-custom-token-${
+                networkError ? 'retry' : 'another'
+              }-btn`}
```

Comment:
I think "add-another" would be a better name for the action.

Reply:
fixed